app/components/SessionTile.jsx

```diff
@@ -0,0 +1,59 @@
+import { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import Component from '../Component';
+import style from './SessionTile.scss';
+import ClassNames from 'classnames';
+import translate from '../i18n/Translate';
+import { Card, CardTitle, CardActions } from 'react-toolbox/lib/card';
+import { default as Button } from 'react-toolbox/lib/button';
+
+const stateToProps = state => ({ });
+
+const actionsToProps = dispatch => ({});
+
+@translate('SessionTile')
+@translate('SessionName')
+@connect(stateToProps, actionsToProps)
+class SessionTile extends Component {
+    renderLastJoined (lastJoined) {
+        const MINUTE = 60 * 1000;
```

Comment:
Yeah I know, but I didn't want to introduce another library for something
so trivial.
Op 12-apr.-2016 12:12 PM schreef "Antoine Jaussoin" <
notifications@github.com>:

> In app/components/SessionTile.jsx
> <https://github.com/antoinejaussoin/retro-board/pull/12#discussion_r59352145>
> :
>
> > +import style from './SessionTile.scss';
> > +import ClassNames from 'classnames';
> > +import translate from '../i18n/Translate';
> > +import { Card, CardTitle, CardActions } from 'react-toolbox/lib/card';
> > +import { default as Button } from 'react-toolbox/lib/button';
> > +
> > +const stateToProps = state => ({ });
> > +
> > +const actionsToProps = dispatch => ({});
> > +
> > +@translate('SessionTile')
> > +@translate('SessionName')
> > +@connect(stateToProps, actionsToProps)
> > +class SessionTile extends Component {
> > +    renderLastJoined (lastJoined) {
> > +        const MINUTE = 60 * 1000;
>
> You could use momentjs if you wanted to simplify this:
> http://momentjs.com/docs/#/displaying/fromnow/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/antoinejaussoin/retro-board/pull/12/files/93a83df3e28a4dbed44a686a6e5f15f054293aad#r59352145>
>